src/registry.json

```diff
@@ -411,6 +411,17 @@
         }
       }
     },
+    "npm:@forta-network/metamask-snap": {
+      "id": "npm:@forta-network/metamask-snap",
```

Comment:
`packageName` is not up to date in this `snap.manifest.json`. Would ask that they update before we add them.

Reply:
OK I have asked them to update

Reply:
OK Forta has published v0.1.2 with the fix